configs/nrel-commute.nrel-op.json

```diff
@@ -27,11 +27,49 @@
             }
         }
     },
+    "survey_info": {
+      "surveys": {
+        "UserProfileSurvey": {
+          "formPath": "json/demo-survey-v2.json",
+          "version": 1,
+          "compatibleWith": 1,
+          "dataKey": "manual/demographic_survey",
+          "labelTemplate": {
+            "en": "Answered",
+            "es": "Contestada"
+          }
+        }
+      },
+      "trip-labels": "MULTILABEL"
+    },
     "display_config": {
         "use_imperial": true
     },
+    "metrics": {
+      "include_test_users": false
+    },
```

Comment:
I added this metrics section because it is the default in other programs such as https://github.com/sebastianbarry/nrel-openpath-deploy-configs/commit/9e62582f6f3e11f6dfa8bba192c6bd5fe33039a7

Do we want to keep this for NREL Commute?